lib/api/build.rs

```diff
@@ -191,9 +191,10 @@ fn configure_validation(builder: Builder) -> Builder {
             ("UpsertPoints.points", ""),
             ("DeletePoints.collection_name", "length(min = 1, max = 255)"),
             ("UpdatePointVectors.collection_name", "length(min = 1, max = 255)"),
-            ("UpdatePointVectors.vectors", "custom(function = \"crate::grpc::validate::validate_named_vectors_not_empty\", message = \"must specify vectors to update\")"),
+            ("UpdatePointVectors.points", ""),
```

Comment:
What?! The build script does not fail on non existing field?

Reply:
Sadly not. It's a limitation in the code generator of tonic/prost.